test/performance/observed_concurency_test.go

```diff
@@ -21,22 +21,27 @@ package performance
 import (
 	"context"
 	"fmt"
+	"net/http"
+	"sort"
+	"strconv"
+	"strings"
+	"sync/atomic"
 	"testing"
 	"time"
 
+	"github.com/pkg/errors"
+
+	pkgTest "github.com/knative/pkg/test"
 	"github.com/knative/pkg/test/logging"
 	"github.com/knative/pkg/test/spoof"
-	"github.com/knative/serving/pkg/apis/serving/v1alpha1"
-	serviceresourcenames "github.com/knative/serving/pkg/reconciler/v1alpha1/service/resources/names"
 	"github.com/knative/serving/test"
-	"github.com/knative/test-infra/tools/prometheus"
 	"github.com/knative/test-infra/tools/testgrid"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"golang.org/x/sync/errgroup"
 )
 
 const (
 	tName       = "TestObservedConcurrency"
-	perfLatency = "perf_latency"
```

Comment:
this string is used by testgrid to show the values on the dashboard. Can we keep it perf_latency?

Reply:
As long as it does not imply "end-user latency" as in: time to send the request until it gets back, then yes!

Reply:
A comment would clarify what this means and where it is used.

Reply:
Also, does it make sense to make it part of performance.go if it's related to the testgrid dashboard?

Reply:
yes. should be moved to performance.go. I can do it once this PR merges